web/pgadmin/static/js/ToolView.jsx

```diff
@@ -38,8 +39,42 @@ ToolForm.propTypes = {
   params: PropTypes.object,
 };
 
+export function getToolTabParams(panelId, toolUrl, formParams, tabParams, restore=false) {
+  let freshTabParams = {    
+    closable: true,
+    manualClose: true,
+    ...tabParams};
+  
+  let restorTabParams = {
+    title: (<TabTitle id={panelId} closable={true} defaultInternal={{...tabParams}}/>),
+    internal: {
+      ...tabParams,
```

Comment:
I think tabParams should be moved after manualClose because it can be true/false. This also says the diff between freshTabParams and restorTabParams is only title.
I think we need to stream line this so that params are consistent. It should not depend on restore or fresh.